packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -624,13 +640,23 @@ export const useNotifiSubscribe: ({
   return {
     resendEmailVerificationLink,
     instantSubscribe,
-    isAuthenticated: client.isAuthenticated,
-    isInitialized: client.isInitialized,
-    isTokenExpired: client.isTokenExpired,
+    isAuthenticated: client?.isAuthenticated ?? false,
+    isInitialized: client?.isInitialized ?? false,
+    isTokenExpired: client?.isTokenExpired ?? false,
```

Comment:
we no longer need the null check on client, right?

Reply:
You are right. addressed.